k8s/admin/jenkins/patches/cftptl/cluster-00/jenkins.yaml

```diff
@@ -193,6 +193,12 @@ spec:
                     name: MI
                     recurse: true
                     title: Management Information Dashboard
+                - buildMonitor:
+                    includeRegex: >-
+                      ^HMCTS_.*PCQ.*\/.+\/master
+                    name: PCQ
+                    recurse: true
+                    title: PCQ Dashboard
```

Comment:
dashboard is implied

```suggestion
                    title: PCQ
```

Reply:
Agree with the suggestion it is implied :)